app/api/submission/TransitOperation.scala

```diff
@@ -31,13 +31,15 @@ object TransitOperation {
 
 object transitOperationType02 {
 
-  private lazy val convertIsSimplifiedProcedure: String => Boolean = {
+  lazy val convertIsSimplifiedProcedure: String => Boolean = {
     case "simplified" => true
     case _            => false
   }
```

Comment:
do we need this val convertIsSimplifiedProcedure?
can you not  have :
  val isSimplifiedReader: Reads[Boolean] = (identificationPath \ "isSimplifiedProcedure").read[String].map(_.equals("simpilified"))